geniza/corpus/tei_transcriptions.py

```diff
@@ -76,9 +76,30 @@ def text_to_html(self):
                 }
             )
 
-        # combine blocks of text into html
+        # combine blocks of text into html, chunked into pages to match sides of images
```

Comment:
It applies when the script is run; you'll probably want to run locally so you can see the changes. I didn't put it in the deploy notes since it's setup to run as a cron script in qa and production, but maybe is worth mentioning so we can run it immediately whenever we deploy this update.